src/SFA.DAS.LoginService.Web/Controllers/ChangeEmail/ConfirmChangeEmailController.cs

```diff
@@ -31,6 +31,7 @@ public IActionResult ConfirmChangeEmail(Guid clientId, [FromQuery] string email,
         [HttpPost("profile/{clientId}/changeemail/confirm")]
         public async Task<IActionResult> ConfirmChangeEmail(Guid clientId, [FromForm] ConfirmChangeEmailViewModel model)
         {
+            model.ClientId = clientId;

```

Comment:
Just a question, and it's perhaps because we have been somewhat isolated in terms of development, but how does setting the model ClientId property here fit into resend?

Reply:
All the controllers in the login service need a client ID somehow - either by looking it up from a `ReturnUrl` querystring, or by being passed it explicitly.

This is because this login service was originally designed to be multi-tenanted, and that somehow spread through the whole website.

Without a client ID, none of the controllers work properly.